packages/core/src/js/feedback/FeedbackWidget.tsx

```diff
@@ -257,7 +257,11 @@ export class FeedbackWidget extends React.Component<FeedbackWidgetProps, Feedbac
     }
 
     const screenshot = getCapturedScreenshot();
-    if (screenshot) {
+    if (screenshot === 'ErrorCapturingScreenshot') {
+      setTimeout(async () => {
+        feedbackAlertDialog(text.errorTitle, text.captureScreenshotError);
+      }, 100);
```

Comment:
The delay is here to let the widget slide back up?

Reply:
Yes 👍